include/depthai-shared/common/CameraFeatures.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "depthai-shared/common/CameraBoardSocket.hpp"
+#include "depthai-shared/common/CameraImageOrientation.hpp"
+#include "depthai-shared/common/CameraSensorType.hpp"
+#include "depthai-shared/utility/Serialization.hpp"
+
+namespace dai {
+
+/**
+ * CameraFeatures structure
+ *
+ * Characterizes detected cameras on board
+ */
+struct CameraFeatures {
+    /**
+     * Board socket where the camera was detected
+     */
+    CameraBoardSocket socket;
+    /**
+     * Camera sensor name, e.g: "IMX378", "OV9282"
+     */
+    std::string sensorName;
+    /**
+     * Maximum sensor resolution
+     */
+    uint16_t width, height;
```

Comment:
```suggestion
    int / std::int32_t width, height;
```

If perhaps someday will be necessary to denote some unknown w/h, etc...